pkg/utils/overrider/overrider.go

```diff
@@ -117,8 +117,8 @@ func FetchAllMatchingOverridesForResourceSnapshot(
 	filteredRO := make([]*placementv1alpha1.ResourceOverrideSnapshot, 0, len(roList.Items))
 	for i := range croList.Items {
 		placementInOverride := croList.Items[i].Spec.OverrideSpec.Placement
-		if placementInOverride != nil && placementInOverride.Name != crp {
-			klog.V(2).InfoS("Skipping this override which was created for another placement", "clusterResourceOverride", klog.KObj(&croList.Items[i]), "placementInOverride", placementInOverride.Name, "clusterResourcePlacement", crp)
+		if placementInOverride != nil && placementInOverride.Name != placementKey {
```

Comment:
Hi Ryan! For now the placementRef struct contains only one field, `Name`; do we plan to add a `Namespace` field as well? Or do we expect users to also use the `namespace/name` format?

Reply:
I am not sure if this comment has been posted correctly; sorry if it is showing up twice.